CONTRIBUTING.md

```diff
@@ -6,11 +6,15 @@ Contributions to this project are [released](https://help.github.com/articles/gi
 
 This project adheres to the [Contributor Covenant Code of Conduct](http://contributor-covenant.org/). By participating, you are expected to uphold this code.
 
-The majority of contributions won't need to touch any Ruby code at all.
```

Comment:
I see you first forked Linguist in [November 2015](https://github.com/github/linguist/pull/4066/commits) (!!). You're overwriting unrelated changes that have been made to the docs since then. Please update `CONTRIBUTING.md` with the [latest version](https://github.com/github/linguist/blob/4f0f9bd51d9bf54e05d1b1456dfdc3df54b3360d/CONTRIBUTING.md).

Reply:
@JJ You might want to update your fork...

Reply:
I'll do that :+1:

Reply:
I've restored the HEAD copy. In fact, advice to install CMAKE, which was the only actual change, is already in the README.md file.